### PR TITLE
Information about testnet, infura, metamask

### PR DESCRIPTION
**Pull request by AyushyaChitransh**
Updated some sections:

 - What is testnet?
 - Using Testnets
 - MetaMask and Testnets
 - Network Provider Infura

I've tried to keep the details short and concise. Feel free to make/suggest corrections.

### CODE REVIEW

File: ethereum-testnets.asciidoc [hunk 1]
```diff
@@ -2,12 +2,31 @@
 
 === What is testnet?
 
+A test network is used to simulate behavior of ethereum network. There are some publicly available test networks.
```
Comment (come-maiz) [inline ethereum-testnets.asciidoc]: I would add articles. Maybe: simulate THE behavior of THE main ethereum network and some variations.

File: ethereum-testnets.asciidoc [hunk 2]
```diff
@@ -2,12 +2,31 @@
...
+When any major change is about to be included in main ethereum network, its tests are mostly done on these test networks.
```
Comment (come-maiz) [inline ethereum-testnets.asciidoc]: in THE main ethereum...

File: ethereum-testnets.asciidoc [hunk 3]
```diff
@@ -2,12 +2,31 @@
...
+These test networks are also used by large number of developers for testing their applicactions before deploying them on main network.
```
Comment (come-maiz) [inline ethereum-testnets.asciidoc]: by A large.

File: ethereum-testnets.asciidoc [hunk 4]
```diff
@@ -2,12 +2,31 @@
...
 === What is testnet?
 
+A test network is used to simulate behavior of ethereum network. There are some publicly available test networks.
+When any major change is about to be included in main ethereum network, its tests are mostly done on these test networks.
+These test networks are also used by large number of developers for testing their applicactions before deploying them on main network. 
+
 === Using Testnets
 
+You can either connect to publicly available test networks or spawn a development network of your own.
+First lets use a public testnet for an easier setup. For using public testnet, you require some testnet ethers
```
Comment (come-maiz) [inline ethereum-testnets.asciidoc]: lets -> let's
To use a public testnet

**Review by come-maiz — CHANGES_REQUESTED** [review]

**Review by fjrojasgarcia — APPROVED** [review]
IMHO sounds fine.

**Pull request merged**